node/narwhal/tests/common/primary.rs

```diff
@@ -76,7 +79,7 @@ impl TestValidator {
                 for connection in connections {
                     debug!("  {}", connection);
                 }
-                sleep(Duration::from_secs(10)).await;
+                sleep(Duration::from_secs(5)).await;
```

Comment:
perhaps this could be a part of `TestNetworkConfig`? `fire_cannons` could be changed from `bool` to `Option<u64>`, indicating the interval that goes here

Reply:
This is the connection timeout but if you mean the cannon sleep, I'm onboard 👍